unitary/alpha/quantum_world.py

```diff
@@ -526,6 +557,103 @@ def get_binary_probabilities(
             binary_probs.append(1 - one_probs[0])
         return binary_probs
 
+    # TODO(pengfeichen): make this function work for compile_to_qubits==True.
+    def get_binary_probabilities_from_state_vector(
```

Comment:
I'm pretty uncomfortable adding this.  Nothing else in the alpha library assumes that we are using a simulator that can create a state vector.  Why do we need this anyway?  Is it because get_histogram_with_all_objects_as_key is too slow and / or needs too many samples?

Reply:
Hi Doug, I was adding this so that we have a choice to access the accurate/expected probability of quantum objects within a quantum world, while the current method is based on sampling and the the returned probability varies call by call.

I did not compare in terms of how fast this implementation vs sampling.

If you don't think it's useful, I can remove it. Thanks.

Reply:
Hi Doug, after some discussion this morning in the meeting, I've removed this util function. Thanks.